junit-jupiter-engine/src/test/java/org/junit/jupiter/api/extension/ExtensionComposabilityTests.java

```diff
@@ -95,12 +95,9 @@ void ensureJupiterExtensionApisAreComposable() {
 
 		// 6) Verify our expectations
 
-		// @formatter:off
-		assertAll(
-				() -> assertThat(actualMethodSignatures).isEqualTo(expectedMethodSignatures),
-				() -> assertThat(actualMethodNames).isEqualTo(expectedMethodNames)
-		);
-		// @formatter:on
+		assertAll("Checking method signatures and names",
+			() -> assertLinesMatch(expectedMethodSignatures, actualMethodSignatures),
+			() -> assertLinesMatch(expectedMethodNames, actualMethodNames));
```

Comment:
Why did you change the assertions here?

Reply:
`assertLinesMatch` provides a better reporting layout. But as above, I will revert all changes made to `ExtensionComposabilityTests.java` in this PR.